test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
```

Comment:
We don't use test folder.
Open `.gitignore` and add `test` to it.
Run `git rm --cached -r test` to untrack it

Reply:
![image](https://user-images.githubusercontent.com/65896888/91243520-654d2a80-e774-11ea-819c-862cc4d27cf9.png)